### DISCUSSION TITLE
Hunger Preview linked with Custom Hunger Icon

### DISCUSSION THREAD

**Discussion by Leooo1702** [BedrockTweaks/Files#196] [Ideas]
When choosing a Custom hunger icon, for example a cookie,  make the Hunger Preview icon a cookie as well, same for the other variants

**Comment by PAPISOP** [comment]
this will be done when we are doing combination update

**Comment by SquatchHunter** [comment]
~~I will start to look into this for next major PR - I was going to look at fixing the Hunger Preview pack anyway~~
afaik PAPI took care of this
